pkg/init/backend/interactive.go

```diff
@@ -220,18 +218,24 @@ func PrintConfiguration(config asker.DevfileConfiguration) {
 
 	for _, key := range keys {
 		container := config[key]
-		color.Green("Container %q:", key)
-		color.Green("  Opened ports:")
+		log.Sectionf("Container Configuration %q:", key)
 
-		for _, port := range container.Ports {
-			color.New(color.Bold, color.FgWhite).Printf("   - %s\n", port)
+		fmt.Printf("  OPEN PORTS:\n")
+
+		for _, value := range container.Ports {
+			fmt.Printf("    - %s\n", value)
 		}
 
-		color.Green("  Environment variables:")
+		fmt.Println("  ENVIRONMENT VARIABLES:")
+
 		for key, value := range container.Envs {
-			color.New(color.Bold, color.FgWhite).Printf("   - %s = %s\n", key, value)
+			fmt.Printf("    - %s = %s\n", key, value)
 		}
+
 	}
+
+	// Make sure we add a newline at the end
+	fmt.Println()
```

Comment:
Please don't use `fmt.Printf` or `fmt.Println` in the business layer. It should ideally be used only in CLI layer. Use `fmt.Fprintf` with stdout as `io.Writer`.